site/oauth2-examples.md

```diff
@@ -86,12 +88,14 @@ Run the following 3 commands to get the environment ready to see Oauth 2.0 plugi
 
 The last command starts a RabbitMQ server with [this](https://github.com/rabbitmq/rabbitmq-oauth2-tutorial/blob/main/conf/uaa/rabbitmq.config) configuration file.
 
+## <a id="access-management-ui" class="anchor" href="#access-management-ui">Access Management UI using OAuth 2.0 tokens</a>
 
-## <a id="use-access-tokens" class="anchor" href="#use-oauth-tokens">Use access tokens</a>
+The Management UI can be configured with one of these two login modes:
 
-The following subsections demonstrate how to use access tokens with any messaging protocol and also to access the management ui and rest api.
+* [Service-Provider initiated logon](#service-provider-initiated-logon) - This is the default and traditional OAuth 2.0 logon mode. The user comes to the Management UI and clicks on the button "Click here to logon" which initiates the logon. The logon process starts in RabbitMQ, the *Service Provider*.
+* [Identity-Provider initiated logon](#identity-provider-initiated-logon) - This is a logon mode meant for web portals. Users navigate to RabbitMQ with a token already obtained by the web portal on behalf of the user.
```

Comment:
So avoid using past tense and use present where possible in documentation - How about: "You can use this logon mode for web portals. Users can navigate to RabbitMQ with an existing token which is retrieved from the web portal on behalf of the user.

I don't really know what the final sentence above means in the context of "Users can...." and then "on behalf of the user" so are you saying a "user can navigate to RabbitMQ with an existing token which is retrieved from the web portal and they can use that token to logon" so it is the user that is navigating that is actually logging on?? If so how about this for a reword: "You can navigate to RabbitMQ with an existing token which you previously retrieved from the web portal, you can then use this token to logon,"????